src/configureStore.tsx

```diff
@@ -13,7 +21,7 @@ export type ApplicationState = {
   example: ExampleState;
 };
 
-const createRootReducer = () => {
+const createRootReducer = (): Reducer<ApplicationState> => {
```

Comment:
TS can already infer this from `return combineReducers<ApplicationState>`. Did you need this for something? If I remove it, I don't get any lint errors.

Reply:
No error but a warning:

```
  26:27  warning  Missing return type on function      @typescript-eslint/explicit-function-return-type
```